hybridq/architecture/plot.py

```diff
@@ -39,8 +41,9 @@ def plot_qubits(qpu_layout: QpuLayout,
                 layout: list[Coupling] = None,
                 subset: list[Qubit] = None,
                 selection: list[Qubit] = None,
-                figsize: tuple[int, int] = (6, 6),
-                draw_border: bool = True,
+                scale: float = 6,
+                figsize: tuple[int, int] = None,
+                draw_border: bool = False,
                 title: str = None) -> None:
```

Comment:
I suggest you return the figure from these plot functions in case the user wants to manipulate the figure further